bite-size-notes/week-8-b-probability-review.md

```diff
@@ -51,6 +54,50 @@ $$P(x,y,z) = P(x \mid y,z) P(y,z) = P(x \mid y,z) P(y \mid z) P(z)$$
 
```

Comment:
Can you provide some additional examples about Chain Rule?

Reply:
I think the two examples are sufficient for the review! I'm not sure how to add more examples without going to the chain rule with 4 RVs.